### ISSUE TITLE
Crash on selecting 'mood'.

### ISSUE THREAD

**Issue by SeerSorin** [henry232323/Pesterchum-Discord#9]
GIF: https://imgur.com/P63UDFZ

No matter which theme I'm in, or what account, selecting a mood seems to crash the client.

**Comment by henry232323** [comment]
I'm not able to reproduce this, can I get you to run the program from the command line

**Comment by SeerSorin** [comment]
I'm not too sure on what I'm supposed to be looking for when I do that (or if I'm doing it wrong, just let me know, I'm not too smart :Y)
GIF: https://imgur.com/XfpWoJ6

**Comment by SeerSorin** [comment]
In addition to this is that if I leave myself on the Trollian theme, it will just crash instantly if I try to go to options.

**Comment by henry232323** [comment]
Oh sweet

**Comment by SeerSorin** [comment]
Yeah, it's pretty awesome, have to delete my options.cfg to get back in

**Comment by henry232323** [comment]
Can I get a gif of you opening it from the command line and switching your mood?

**Comment by SeerSorin** [comment]
When you say opening from the command line, do you mean like

GIF: https://imgur.com/IyJDH0b

Or am I just _**really really stupid**_? (I probably am.)

**Comment by henry232323** [comment]
Yeah thats it

**Comment by henry232323** [comment]
Can you try waiting for it to login before you click the emote buttons?

**Comment by SeerSorin** [comment]
> 
> 
> Can you try waiting for it to login before you click the emote buttons?

The same thing happens, even when I go into a memo and send a message first.
